dashboard/installation/pipelines/dashboard.release.yaml

```diff
@@ -49,7 +49,7 @@ stages:
             displayName: Download Artifact
           - template: ./Templates/dashboard-release-steps.yaml
             parameters:
-              azureSubscription: $(Infra.Environment.ServiceConnection)
+              azureSubscription: 'YourAzureSubscription'
```

Comment:
NameOfYourServiceConnection is better I think

Reply:
Changed in [`e718f9d` (#138)](https://github.com/invictus-integration/docs-ifa/pull/138/commits/e718f9d3a6d85baabe8033c134d1e69d4b502033)